src/main/java/org/synyx/urlaubsverwaltung/person/PersonServiceImpl.java

```diff
@@ -60,6 +60,8 @@ public Person create(String username, String lastName, String firstName, String
         accountInteractionService.createDefaultAccount(person);
         workingTimeService.createDefaultWorkingTime(person);
 
+        applicationEventPublisher.publishEvent(new PersonCreatedEvent(persistedPerson.getId()));
```

Comment:
Maybe extract proxy/decorator PersonServiceMetricsImpl or use aspect for metric expose